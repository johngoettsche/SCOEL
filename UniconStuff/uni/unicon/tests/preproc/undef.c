/* test #undef */

#undef __RCRS__
#define x 3
#begdef f(a,b)
a+b
#enddef
#undef x
#undef f
__RCRS__
x
f(1,2)
