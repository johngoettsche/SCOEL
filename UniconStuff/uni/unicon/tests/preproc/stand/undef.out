#line 9 "undef.c"

__RCRS__
x
f(1,2)
